.github/workflows/staging-workflow.yml

```diff
@@ -7,7 +7,6 @@ on:
     paths:
```

Comment:
Probably need to rebase the latest from master, but there is now a production-workflow.yml file which has geo-service in it. We should remove it from that file as well.

Reply:
Done (locally so far). One thing I forgot we need to add GOOGLE_MAPS_API_KEY. Terraform file wise it would only be to add here, correct `main.tf`?

```
    {
      name  = "GOOGLE_MAPS_API_KEY"
      value = data.aws_ssm_parameter.google_maps_api_key.value
    },
```

I'll get you the secret in slack to store.

Reply:
We planned to remove REACT_APP_AIRTABLE keys & references in front-end but there wasn't another PR so I did it here - f3d5aaec292219118ae33a7bd581d2ea4ce88013 

So I also removed it in production-workflow.yml f3d5aaec292219118ae33a7bd581d2ea4ce88013

Let me know if the looks okay

Reply:
Looks good. You should also remove it in the other workflows as well